rpm/update-config-files.sh

```diff
@@ -57,7 +57,7 @@ fi
 
 # Set these on Fedora Rawhide (41+), RHEL 10+, and on all COPR builds
 # regardless of distro
```

Comment:
Please backport the comment update as well, so that it is consistent with the code.

LGTM otherwise.

Reply:
Also included changes from #2213. PTAL